projects/schematics/src/shared/constants.ts

```diff
@@ -1049,6 +1051,7 @@ export const EXTRACT_PRODUCT_CARD_PARAMETERS = 'extractProductCardParameters';
 export const CONFIGURATOR_API_IS_ATTRIBUTE_GROUP = 'isAttributeGroup';
 export const CONFIGURATOR_API_GET_CONFLICT_MESSAGE_KEY =
   'getConflictMessageKey';
+export const CONVERT_ATTRIBUTE_TYPE = 'convertAttributeType';
```

Comment:
Please have 'CONFIGURATOR' as part of this constant name (in order to find all our constants easier)

Reply:
Done!